tests/unit_tests/test_messaging.py

```diff
@@ -401,7 +401,7 @@ def test_connect_passes_basic_auth_details(
     client.connect()
     mock_connection.connect.assert_called_once_with(
         username=authentication.username,
-        passcode=authentication.password,
+        passcode=authentication.password.get_secret_value(),
```

Comment:
I think it will be better if we change this to password as well ...like in [blueapi](https://github.com/DiamondLightSource/blueapi/blob/8b35390a8a4fd1a39ea573a325b568ce5450dbd4/tests/unit_tests/test_config.py#L133)

Reply:
This is in the stomp.py library that we are using, so we can't. This is adapting the change from passcode -> password everywhere that we can.
https://jasonrbriggs.github.io/stomp.py/api.html#establishing-a-connection
https://stomp.github.io/stomp-specification-1.2.html#CONNECT_or_STOMP_Frame